.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: dart
 dart_task:
+- test: --platform vm
```

Comment:
Consider combining these? Booting up and running `pub get` costs a minute for each run.

Reply:
What would the syntax of that be? I can't see anything useful on https://docs.travis-ci.com/user/languages/dart/

Reply:
`--platform vm,chrome`